src/plugin.js

```diff
@@ -133,7 +133,7 @@ const initPlugin = function(player, options) {
   };
 
   const onPlayStartMonitor = function() {
-    let lastTime = 0;
+    let lastTime = player.currentTime();
```

Comment:
Previously a `play` after an error was throw would cause this to trigger and `lastTime` would be reset to 0. This would then cause the check below to always call `resetMonitor()`. With this change `lastTime` will reset to `currentTime` and `resetMonitor()` will only be called if we know that playback has actually started, which was the original intention.